drivers/net/ethernet/intel/idpf/idpf_eth_common.h

```diff
@@ -0,0 +1,254 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+/* Copyright (C) 2023 Intel Corporation */
+
+#ifndef _IDPF_ETH_COMMON_H_
+#define _IDPF_ETH_COMMON_H_
+
+#define IDPF_LARGE_MAX_Q			256
+/* Mailbox Queue */
+#define IDPF_MAX_MBXQ				1
+
+#define GETMAXVAL(num_bits)		GENMASK((num_bits) - 1, 0)
+#define IDPF_MAX_BUFQS_PER_RXQ_GRP		2
+
+#define IDPF_TX_MIN_PKT_LEN		17
```

Comment:
should be part of aux driver headers. maybe idpf_txrx.h

Reply:
done!